lib/videoroom/application.ex

```diff
@@ -11,6 +11,11 @@ defmodule Videoroom.Application do
   def start(_type, _args) do
     RoomRegistry.create()
 
+    # credo:disable-for-next-line Credo.Check.Warning.MixEnv
+    if Mix.env() == :test do
+      Divo.Suite.start(services: [:jellyfish], auto_start: false)
+    end
```

Comment:
That's definitely not perfect but it at least somehow works.

The problem is that it seems that divo tries to cleanup docker images before RoomService exits so we turn Jellyfish off, WS connection goes down and our RoomService crashes because it's linked to the Notifier, which results in a bunch of error logs. If we want to make it better, let's do this in another PR. 


I can also entirely remove divo and make an rename mix integration_test to mix test so that we always test using docker compose.